notebooks/signac_202_Integration_with_pandas.ipynb

```diff
@@ -6,6 +6,14 @@
    "source": [
```

Comment:
<p>I can't tell on reviewnb.com, but can you add a hyperlink to the method reference in the doc (if it doesn't do that already)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/signac-examples/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>